pando-rt/src/start.cpp

```diff
@@ -90,11 +98,18 @@ extern "C" int __start(int argc, char** argv) {
               break;
           }
         }
-        if(task.has_value()) { (*task)(); task = std::nullopt; }
-        else {
+        if(task.has_value()) {
+          (*task)();
+          task = std::nullopt;
+        } else {
           counter::recordHighResolutionEvent(idleCount, idleTimer, false, thisPlace.core.x, coreDims.x);
         }
+#ifdef PANDO_RT_USE_BACKEND_PREP
       } while (*coreActive == true);
+#elif defined(PANDO_RT_USE_BACKEND_DRVX)
+        pando::hartYield();
+      } while (!getTerminateFlag());
```

Comment:
why can't this go inside the operator* inside the coreActive

Reply:
Done